internal/cmd/common.go

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (C) 2021 Canonical Ltd
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0'
+ */
+
+package cmd
+
+import (
+	"github.com/edgexfoundry/edgex-cli/internal/config"
+	"github.com/edgexfoundry/edgex-cli/internal/format"
+	"github.com/spf13/cobra"
+)
+
+var verbose, metadata, data, command, notification, scheduler, json bool
+
+func getSelectedService() string {
+	if metadata {
+		return "core-metadata"
```

Comment:
```suggestion
		return common.CoreMetaDataServiceKey
```
Same for other cases
Constants are here: https://github.com/edgexfoundry/go-mod-core-contracts/blob/main/common/constants.go#L256-L268

Reply:
thanks. all updated now.